core/src/test/java/org/elasticsearch/index/query/TermsQueryBuilderTests.java

```diff
@@ -200,9 +201,9 @@ public void testBothValuesAndLookupSet() throws IOException {
                 "}";
         try {
             parseQuery(query);
-            fail("Expected IllegalArgumentException");
-        } catch(IllegalArgumentException e) {
-            assertThat(e.getMessage(), is("Both values and termsLookup specified for terms query"));
+            fail("Expected ParsingException");
+        } catch(ParsingException e) {
+            assertThat(e.getMessage(), containsString("[terms] query does not support more than one field."));
```

Comment:
You switched to using the query builder NAME in other places - why not here as well?